lua/plenary/curl.lua

```diff
@@ -243,8 +243,13 @@ end
 
 local request = function(specs)
   local response = {}
+  local compressed = true
+  local is_windows = vim.loop.os_uname().sysname == "Windows_NT"
```

Comment:
i dont like how we check for windows here. its easier to check using this here `package.config:sub(1,1) == "\\"`

Reply:
Okay, didn't that way existed. Thank you for the comment.

I will change it right now 🫡